packages/web/src/scss/components/Skeleton/_Skeleton.scss

```
@@ -0,0 +1,90 @@
+@use 'theme';
+@use 'tools';
+@use '@tokens' as tokens;
+@use '../../tools/breakpoint';
+@use '../../tools/responsive-properties';
+@use '../../tools/string' as spirit-string;
```

```suggestion
@use '@tokens' as tokens;
@use '../../tools/breakpoint';
@use '../../tools/responsive-properties';
@use '../../tools/string' as spirit-string;
@use 'theme';
@use 'tools';
```

we sort by path, please. (we try to. I know this is not everywhere correct; we try to fix it as we see it) 

The order is global sass modules (@use 'sass:*'), loaded modules (@tokens) and the rest by the path from the furthest to the nearest.